docker/docker-compose-base.yml

```diff
@@ -1,13 +1,15 @@
 services:
     app:
-      image: ghcr.io/openboxes/openboxes:feature-upgrade-to-grails-3.3.10
+      image: ${OB_IMAGE_REPOSITORY-ghcr.io/}openboxes/openboxes:${OB_VERSION:-latest}
       container_name: openboxes-app
       expose:
         - "8080"
+      ports:
+        - 8080:8080
```

Comment:
this was a "bug" that I fixed. By properly exposing the container ports, we can now hit the server at `http://localhost:8080/openboxes` from outside the container